model/patch/db.go

```diff
@@ -139,14 +140,58 @@ func ByGithash(githash string) db.Q {
 
 type ByPatchNameStatusesCommitQueuePaginatedOptions struct {
 	Author             *string
-	Project            *string
-	PatchName          string
-	Statuses           []string
-	Page               int
-	Limit              int
 	IncludeCommitQueue *bool
 	IncludeHidden      *bool
+	Limit              int
 	OnlyCommitQueue    *bool
+	Page               int
+	PatchName          string
+	Project            *string
+	Requesters         []string
+	Statuses           []string
+}
+
+// Based off of the implementation for Patch.GetRequester.
+var requesterExpression = bson.M{
+	"$switch": bson.M{
+		"branches": []bson.M{
+			{
+				"case": bson.M{
+					"$and": []bson.M{
+						{"$ifNull": []interface{}{"$" + githubPatchDataKey, false}},
+						{"$ne": []string{"$" + bsonutil.GetDottedKeyName(githubPatchDataKey, "head_owner"), ""}},
```

Comment:
nit: rather than hard-code the BSON key, use a variable like the other `bsonutil.MustHaveTag` keys.